src/test/FuzzerImpl.cpp

```diff
@@ -69,18 +70,33 @@ getShortKey(PublicKey const& pk)
     return getShortKey(pk.ed25519());
 }
 
+void
+setAssetCode4(AssetCode4& code, int digit)
+{
+    strToAssetCode(code, "Ast" + std::to_string(digit));
+}
+
 // constructs an Asset structure for an asset issued by an account id comprised
 // of bytes reading [0,0,...,digit] and an alphanum4 asset code of "Ast + digit"
 Asset
 makeAsset(int digit)
 {
     Asset asset;
     asset.type(ASSET_TYPE_CREDIT_ALPHANUM4);
```

Comment:
I agree -- I think it could use more discussion as to how we generate valid pseudorandom `AssetCode`s, in particular because we also do so in places such as `makeValid(OfferEntry&)` but in a very limited way.  So I filed #2924 for that one, and mentioned that an improvement there could be used here in fuzzing as well.  If you're okay with making that a future PR, that'd be my preference.

Reply:
Yeah that sounds good to me. Thanks!